### PR TITLE
Correct maker:Ricoh and add GR II

### PR DESCRIPTION
**Pull request by teemodk**
Ricoh GR2 profile data added, but the profile was not detected for my Ricoh GR II.
Changing maker to "RICOH IMAGING COMPANY, LTD." fixed the detection.

### CODE REVIEW

File: data/noiseprofiles.json [hunk 1]
```diff
@@ -6064,7 +6064,7 @@
       ]
     },
     {
-      "maker": "Ricoh",
```
Comment (schenlap) [inline data/noiseprofiles.json]: This breaks all other Ricoh cameras

File: data/noiseprofiles.json [hunk 2]
```diff
@@ -6096,6 +6096,37 @@
...
+            {"name": "GR II iso 10000", "iso": 10000, "a": [0.000470754538385169, 0.000278490538041487, 0.00106726186680657], "b": [2.19023981442033e-06, 1.44319831387867e-06, 2.72566693533878e-06]},
+            {"name": "GR II iso 12800", "iso": 12800, "a": [0.000654284461393812, 0.000388331036020567, 0.00140388667581021], "b": [3.16081319143953e-06, 2.10091677669883e-06, 3.61077876740858e-06]},
+            {"name": "GR II iso 16000", "iso": 16000, "a": [0.000824991340066458, 0.000501822035505979, 0.00177609313840165], "b": [4.19575226573771e-06, 2.76312184961383e-06, 4.82022236844907e-06]},
+            {"name": "GR II iso 20000", "iso": 20000, "a": [0.00101678286005032, 0.000616390668488325, 0.00223308013397197], "b": [5.97549949265273e-06, 4.07637139131783e-06, 6.64485549471867e-06]},
+            {"name": "GR II iso 25600", "iso": 25600, "a": [0.00142158488506293, 0.000890620215779999, 0.00278443118012513], "b": [7.80690184138186e-06, 5.38540540423255e-06, 8.35365270915029e-06]}
+          ]
```
Comment (schenlap) [inline data/noiseprofiles.json]: please create a ticket on https://redmine.darktable.org/projects/darktable/issues/new and add dt-noiseprofile-*.tar.gz
of noise profile. This is necessary to check the profile.

File: data/noiseprofiles.json (hunk 1 above)
Comment (teemodk) [reply data/noiseprofiles.json]: I don't think so, there is only GR besides this new GR II. As far as I know the maker entry is the same for both cameras. EXIF say "RICOH IMAGING COMPANY, LTD."
Are you sure the GR was ever tested?

File: data/noiseprofiles.json (hunk 2 above)
Comment (teemodk) [reply data/noiseprofiles.json]: I will do this. Am currently working on a better finetuned profile.

File: data/noiseprofiles.json (hunk 1 above)
Comment (teemodk) [reply data/noiseprofiles.json]: In case GR need the simple "Ricoh" header, I will have to add above as a separate entry, is that acceptable?

**Comment by schenlap** [comment]
Please amend your commits to avoid a non buildable commit.

**Comment by teemodk** [comment]
Since custom (modified) profiles are not accepted, which is fair enough, I will just close this. Sorry for the time lost.

**Pull request closed without merging**